fdbclient/SpecialKeySpace.actor.cpp

```diff
@@ -677,17 +692,12 @@ Future<RangeResult> ConflictingKeysImpl::getRange(ReadYourWritesTransaction* ryw
                                                   GetRangeLimits limitsHint) const {
```

Comment:
Let's revert this so that we have consistency across all the release branches

Reply:
Done. Thanks.